ArduPlane/Parameters.cpp

```diff
@@ -637,35 +637,34 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Advanced
     GSCALAR(scaling_speed,        "SCALING_SPEED",    SCALING_SPEED),
 
-    // @Param: MIN_GNDSPD_CM
+    // @Param: MIN_GROUNDSPEED
     // @DisplayName: Minimum ground speed
     // @Description: Minimum ground speed in cm/s when under airspeed control
-    // @Units: cm/s
+    // @Units: m/s
     // @User: Advanced
-    ASCALAR(min_gndspeed_cm,      "MIN_GNDSPD_CM",  MIN_GNDSPEED_CM),
+    ASCALAR(min_groundspeed,      "MIN_GROUNDSPEED",  MIN_GROUNDSPEED),
 
-    // @Param: TRIM_PITCH_CD
+    // @Param: TRIM_PITCH_DEG
     // @DisplayName: Pitch angle offset
-    // @Description: Offset applied to AHRS pitch used for in-flight pitch trimming. Correct ground leveling is better than changing this parameter.
-    // @Units: cdeg
-    // @Range: -4500 4500
-    // @Increment: 10
-    // @User: Advanced
-    GSCALAR(pitch_trim_cd,        "TRIM_PITCH_CD",  0),
+    // @Description: Offset in degrees used for in-flight pitch trimming for level flight. Correct ground leveling is an alternative to changing this parameter.
+    // @Units: deg
+    // @Range: -45 45
+    // @User: Standard
+    GSCALAR(pitch_trim,             "TRIM_PITCH_DEG",  0.0f),
 
-    // @Param: ALT_HOLD_RTL
+    // @Param: RTL_ALTITUDE
```

Comment:
Should be RTL_ALT because consistency generally but also specifically Q_RTL_ALT.

Reply:
i prefer RTL_ALTITUDE

Reply:
I'm with Tim on this one, `RTL_ALT` is consistent with both copter and quadplane.

Reply:
except RTL_ALT in copter is in cm, so using the same name is actually a disadvantage. I think that when copter changes over to meters like plane is doing now it would be good for copter to rename to RTL_ALTITUDE

Reply:
`Q_RTL_ALT` is in meters. We could it to `Q_RTL_ALTITUDE`. I don't mind what we end up with, I just want consistency.

Reply:
RTL_ALTITUDE sounds good to me.